test/__fixtures/utils/happn-test-helper.js

```diff
@@ -0,0 +1,54 @@
+const happn = require('../../../..');
+const service = happn.service;
+
+class HappnTestHelper {
+  constructor(serverConfig) {
+    this.serverConfig = serverConfig;
+  }
+
+  static create(serverConfig) {
+    return new HappnTestHelper(serverConfig);
+  }
+
+  async createService() {
+    return new Promise((resolve, reject) => {
+      service.create(this.serverConfig, (e, happnInst) => {
+        if (e) return reject(e);
+        resolve(happnInst);
+      });
+    });
+  }
+
+  async createLocalClient() {
```

Comment:
What is the purpose of async here when there is no await used in the code within it.

Reply:
removing in next commit - retesting locally